organize/models.py

```diff
@@ -182,7 +183,7 @@ class Coorganizer(models.Model):
         related_name="coorganizers")
     email = models.EmailField()
     first_name = models.CharField(max_length=30)
-    last_name = models.CharField(max_length=30)
+    last_name = models.CharField(max_length=30, blank=True)
```

Comment:
Also if we permit `blank=True`, and we don't want `null=True` (as this is in #83 ) we should add `default=""` here.